### PR TITLE
fix(pluralize): correctly pluralize for english language

### PR DESCRIPTION
**Pull request by ValeraS**

### CODE REVIEW

File: src/pluralize.ts [hunk 1]
```diff
@@ -1,25 +1,60 @@
...
+    switch (lang) {
+        case 'ru': {
+            return pluralizeRu(keyValue, count);
+        }
+        case 'en': {
+            return pluralizeEn(keyValue, count);
+        }
+    }
+
+    throw new Error(`Can't pluralize for unknown language: ${lang}`);
```
Comment (ogonkov) [inline src/pluralize.ts:13]: This is a breaking change

Comment (ValeraS) [reply src/pluralize.ts:13]: Let's make major release.

Comment (ogonkov) [reply src/pluralize.ts:13]: May be we can fallback to english in this case?

Comment (ValeraS) [reply src/pluralize.ts:13]: But in many cases it won't be right

**Comment by ierehon1905** [comment]
Ping
@ogonkov @ValeraS

**Pull request closed without merging**